packages/bp/e2e/admin/bots.test.ts

```diff
@@ -18,58 +18,57 @@ describe('Admin - Bot Management', () => {
   const importBotId = 'import-bot'
   const workspaceId = 'default'
 
-  const clickButtonForBot = async (buttonId: string, botId: string) => {
-    const botRow = await expectMatchElement('.bp_table-row', { text: botId })
-    await clickOn('#btn-menu', undefined, botRow)
-    await clickOn(buttonId, undefined)
-  }
-
   beforeAll(async () => {
-    await loginIfNeeded()
+    await loginOrRegister()
     await gotoAndExpect(`${bpConfig.host}/admin/workspace/${workspaceId}/bots`)
   })
 
   it('Import bot from archive', async () => {
-    await page.waitFor(200)
+    await page.waitForSelector('#btn-create-bot')
     await clickOn('#btn-create-bot')
```

Comment:
Instead of duplicating those 2 lines all the time, shouldn't we just add the waitForSelector to the clickOn method ? I dont see any situation where we don't want to wait for the selector to be present before clicking

Reply:
Good point, I will make those changes! :muscle: